cachito/workers/tasks/general.py

```diff
@@ -173,4 +173,35 @@ def filter_git_dir(tar_info):
         # Add the dependencies to the bundle
         bundle_archive.add(str(bundle_dir.deps_dir), "deps")
 
+
+def aggregate_packages_data(request_id: int) -> None:
+    """Aggregate packages data generated for each package manager into one unified data file.
+
+    :param int request_id: the request id.
+    """
+    set_request_state(request_id, "in_progress", "Aggregating packages data")
+    bundle_dir: RequestBundleDir = RequestBundleDir(request_id)
+
+    def _get_packages_data():
+        request = get_request(request_id)
+        for pkg_manager in request["pkg_managers"]:
+            data = PackagesData()
+            data.load(getattr(bundle_dir, f"{pkg_manager}_packages_data"))
```

Comment:
This will not work for `git-submodule`. It might be good to add a test case for that